src/serialization/pkl2proto.py

```diff
@@ -0,0 +1,136 @@
+import pickle
```

Comment:
On second glance, it looks like there's a lot of duplicated code between this file and `protoBufTest.py`. Why is that? Can we make do with a single file so we don't have code bloat?

Reply:
Resolved.